src/Main.java

```diff
@@ -0,0 +1,137 @@
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        Main obj = new Main();
+
+        System.out.println("Метод findSymbolOccurance");
+        System.out.println("Символ зустрічається - " + obj.findSymbolOccurance("Java", 'a') + " раз/разів");
+
+        System.out.println();
+
+        System.out.println("Метод findWordPosition");
+        System.out.println("Result : " + obj.findWordPosition("Apollo", "pollo" ));
+        System.out.println("Result : " + obj.findWordPosition("Apple", "Plant" ));
+
+        System.out.println();
+
+        System.out.println("Метод stringReverse");
+        System.out.println("Result : " + obj.stringReverse("Hello"));
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        System.out.println(obj.isPalindrome("ERE"));
+        System.out.println(obj.isPalindrome("Allo"));
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+    }
+
+    public int findSymbolOccurance(String str, char ch) {
+
+        char[] letters = str.toCharArray();
+        int count = 0;
+
+        for ( int i = 0; i < letters.length; i++ ) {
+            if ( letters[i] == ch ) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public int findWordPosition(String source, String target) {
+
+        int index = source.indexOf(target);
+        if (index != -1) {
+            return index;
+        } else {
+            return -1;
+        }
+    }
+
+    public String stringReverse(String simpleString) {
+
+        char[] stringInLetters = simpleString.toCharArray();
+        String result = "";
+
+        for ( int i = stringInLetters.length - 1; i >= 0; i--) {
+            result += String.valueOf(stringInLetters[i]);
+        }
+        return result;
+    }
+
+    public boolean isPalindrome(String source) {
+
+        String lower = source.toLowerCase();
+        char[] sourceInLetters = lower.toCharArray();
+        String reverseWord = "";
+
+        for ( int i = sourceInLetters.length - 1; i >= 0; i--) {
+            reverseWord += String.valueOf(sourceInLetters[i]);
+        }
+
+        if (lower.equals(reverseWord)) {
```

Comment:
If-else statement can be simplified like this: `return lower.equals(reverseWord)`